dotnet/StachExtensions/FactSet.Protobuf.Stach.Extensions/V2/ColumnOrganizedStachExtension.cs

```diff
@@ -51,32 +50,52 @@ private static Models.Table GenerateTable(Stach.V2.Package package, string prima
                 Metadata = new Dictionary<string, string>()
             };
 
-            var headerDataRowList = headerTable.Data.Rows.Select(x => x.Id).ToList();
-            foreach (var headerTableSeriesDefinition in headerTable.Definition.Columns)
+            if (headerId.Length > 0)
```

Comment:
Can we add similar comment here to easily understand if/else 
https://github.com/factset/stach-extensions/pull/20/files#diff-49b3b99067dbe96b5912ac805fd6ff3282c258f36430ed279477611de0c5065bR89

Reply:
done https://github.com/factset/stach-extensions/pull/19/commits/176725040ed042909b19fce01e1701ced9e2d56d